### DISCUSSION TITLE
Live recording audio and video are out of sync

### DISCUSSION THREAD

**Discussion by liuxiansheng52** [aclap-dev/video-downloadhelper#456] [Help and Q&A]
### Your question

When I use this software to record a live broadcast on the Stripchat website, the file gets stuck and the audio and video are out of sync. But this phenomenon does not occur when recording live broadcasts on other websites. Sometimes, in addition to the audio and video being out of sync and lagging, there will also be automatic disconnection and error reporting. Is this a problem caused by my network? But other websites are downloading at a speed of about 5M, and more than 20 links can be downloaded at one time without lag.

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: zh-CN
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "D:\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Administrator",
  "converterBinary": "D:\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f36095"
}

**Comment by Zorklis** [comment]
same
